definitions/ext-network_dhcp_service/definition.yml

```diff
@@ -0,0 +1,38 @@
+domain: EXT
+type: NETWORK_DHCP_SERVICE
+synthesis:
+  rules:
+  # IPv4 DHCP Service via Gigamon
+  - identifier: dst_ip
+    name: dst_ip
+    conditions:
```

Comment:
all the queries seems to be against `Gigamon` eventType, should we add that into the conditions?

Reply:
Done